app/src/ui/app.tsx

```diff
@@ -1593,6 +1593,11 @@ export class App extends React.Component<IAppProps, IAppState> {
   }
 
   private onRepositoryFilterTextChanged = (text: string) => {
+    if (!/\S/.test(text)) {
```

Comment:
This means I can no longer clear the filter box 😱

I think we wanna address this at a lower level. The selection is happening because of:

https://github.com/desktop/desktop/blob/930e9e8b62beaa2ffb1bae0bac09899c4b994666/app/src/ui/lib/filter-list.tsx#L349-L358

Blame is failing me, but I _think_ the intent was that it would select the first option when filtering. But we shouldn't do that if the filter text is empty.

Reply:
I avoided making the change at the component level because I'm not positive we want this behavior across all usages of the filter list.